obspy/core/event/base.py

```diff
@@ -40,10 +40,41 @@
 
 
 class QuantityError(AttribDict):
-    uncertainty = None
-    lower_uncertainty = None
-    upper_uncertainty = None
-    confidence_level = None
+    """
+    Uncertainty information for a physical quantity.
+
+    :type uncertainty: float
+    :param uncertainty: Uncertainty as the absolute value of symmetric
+        deviation from the main value.
+    :type lower_uncertainty: float
+    :param lower_uncertainty: Uncertainty as the absolute value of deviation
+        from the main value towards smaller values.
+    :type upper_uncertainty: float
+    :param upper_uncertainty: Uncertainty as the absolute value of deviation
+        from the main value towards larger values.
+    :type confidence_level: float
+    :param confidence_level: Confidence level of the uncertainty, given in
+        percent (0-100).
+    """
+    defaults = {"uncertainty": None, "lower_uncertainty": None,
```

Comment:
I don't know either.. in any case that's the current implementation of AttribDict..